front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -165,10 +165,9 @@ export function AgentMessage({
       // Handle validation dialog separately.
       if (eventPayload.data.type === "tool_approve_execution") {
         showValidationDialog({
-          workspaceId: owner.sId,
-          messageId: message.sId,
-          conversationId: conversationId,
-          action: eventPayload.data.action,
+          messageId: eventPayload.data.messageId,
+          conversationId: eventPayload.data.conversationId,
+          actionId: eventPayload.data.actionId,
```

Comment:
Seems like we only need actionId, and it's hard to type action in `output_schema`, so can I just pass around `actionId` ?